tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/observability/tracing/MainFunctionTestCase.java

```diff
@@ -53,8 +53,9 @@ public void testMainMethod() throws Exception {
         final String entryPointFunctionModule = DEFAULT_MODULE_ID;
         final String entryPointFunctionName = "main";
         final List<BMockSpan.CheckPoint> expectedCheckpoints = Arrays.asList(
+                new BMockSpan.CheckPoint(entryPointFunctionModule, FILE_NAME + ":20:5"),
                 new BMockSpan.CheckPoint(entryPointFunctionModule, FILE_NAME + ":22:13"),
-                new BMockSpan.CheckPoint(entryPointFunctionModule, FILE_NAME + ":24:15"),
+                new BMockSpan.CheckPoint(entryPointFunctionModule, FILE_NAME + ":25:23"),
```

Comment:
This is reverting changes done by 
https://github.com/ballerina-platform/ballerina-lang/pull/34374/commits/29899428bcc91807cbd27296c6f6554436f6a5e6